app/models/profile.rb

```diff
@@ -63,7 +63,8 @@ def self.gpo_verification_pending
   end
 
   def self.in_person_verification_pending
-    where.not(in_person_verification_pending_at: nil)
+    where.not(in_person_verification_pending_at: nil).
+      left_outer_joins(:in_person_enrollment).where.not(in_person_enrollment: { status: :expired })
```

Comment:
Same [question](https://github.com/18F/identity-idp/pull/11129/files#r1727757555) for this change.